my/pinboard.py

```diff
@@ -1,21 +1,26 @@
 """
 [[https://pinboard.in][Pinboard]] bookmarks
 """
-from .common import get_files
+REQUIRES = [
+    'git+https://github.com/karlicoss/pinbexport',
+]
 
-from my.config.repos.pinbexport import dal as pinbexport
 from my.config import pinboard as config
 
-# TODO would be nice to make interfaces available for mypy...
+
+import pinbexport.dal as pinbexport
+
 Bookmark = pinbexport.Bookmark
 
 
 # yep; clearly looks that the purpose of my. package is to wire files to DAL implicitly; otherwise it's just passtrhough.
-def dal():
-    sources = get_files(config.export_dir, glob='*.json')
-    model = pinbexport.DAL(sources)
+def dal() -> pinbexport.DAL:
+    from .core import get_files
+    inputs = get_files(config.export_dir) # todo rename to export_path
```

Comment:
+1, standardizing (`export_dir`/`takeout_path`/`export_db` to `export_path`) simplifies the `config`:
https://github.com/karlicoss/HPI/issues/44